go-controller/pkg/ovn/master.go

```diff
@@ -789,12 +787,6 @@ func (oc *DefaultNetworkController) deleteNodeEvent(node *kapi.Node) error {
 		"various caches", node.Name)
 
 	if config.HybridOverlay.Enabled {
```

Comment:
Why don't we need to `oc.lsManager.DeleteSwitch(node.Name)`?

Reply:
Looks like rebase issue from my side.  I have missed some bits of this [commit](https://github.com/numansiddique/ovn-kubernetes/commit/fb62ce7f197e38b82586adca8b5fb27422851dbc) 

Thanks for finding this.  I'll fix it.